config/query-builder.php

```diff
@@ -0,0 +1,44 @@
+<?php
```

Comment:
ik mis nog de configuratie voor de json api. We zullen hem niet zo snel nodig hebben, maar wel altijd handig om hem erin te hebben.

Reply:
5c2b72510558b7ca98b92d69bd4a785c9cb93200